Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -228,6 +229,15 @@ bool PPCSymbolDB::LoadMap(const std::string& filename, bool bad)
   if (!f)
     return false;
 
+  if (!MSR.DR)
+  {
+    PanicAlertT(
```

Comment:
Should we add a `// i18n` comment to explain how this should be translated? I feel like technical terms like MSR are going to be completely translated and possibly end up incomprehensible.

I would also "Failed to load symbol map." or something like that at the beginning because this is supposed to be an error message.

Reply:
The functions that call this throw that very error message when false is returned.

Reply:
Or would you rather the error message be moved inside LoadMap?

Reply:
nvm that would not work for the usage in MIOS.cpp

Reply:
In this case, one of the alerts should be removed because having two panic alerts in a row doesn't make for a very good UX. It's also inconsistent with the other failure paths and adds yet another panic alert into Core code.

I'd get rid of the PanicAlert here and move it into the UI (MenuBar::TryLoadMapFile) -- ModalMessageBox has the additional advantage of not being suppressed even if panic handlers are disabled. This means you would have to return some kind of error code to show the actual failure reason though.

For the other usages of this function, a failure isn't really critical.

Reply:
Is there ever a case where the MSR.DR field will be 0 when booting a game?

Reply:
After emulated BS2, no. Unless the apploader sets MSR.DR to 0, but I doubt any official title would do that.

Reply:
Your suggestions have brought all changes down to just MenuBar.cpp.  I like that :).